moveit_ros/hybrid_planning/README.md

```diff
@@ -0,0 +1,23 @@
+# Hybrid Planning
+Implementation of MoveIt 2's new hybrid planning architecture. You find more information in the project's issues[#300](https://github.com/ros-planning/moveit2/issues/300), [#433](https://github.com/ros-planning/moveit2/issues/433)  and on the [MoveIt 2 roadmap](https://moveit.ros.org/documentation/contributing/roadmap/).
```

Comment:
```suggestion
A hybrid planning architecture. You can find more information in the project's issues[#300](https://github.com/ros-planning/moveit2/issues/300), [#433](https://github.com/ros-planning/moveit2/issues/433)  and on the [MoveIt 2 roadmap](https://moveit.ros.org/documentation/contributing/roadmap/).
```

Reply:
Remove the word "new" because it won't be new for long. Best to keep time-based statements out of the documentation.